docs/notes/getting-started.rst

```diff
@@ -13,6 +13,11 @@ and can be easily installed with :code:`pip`:
 
     pip3 install vineyard
 
+Installing etcd
+-------------------
```

Comment:
Keep the consistence between the `--------` line and the title line, otherwise the sphinx would complain.

Reply:
Get it, thanks for reminding.